### DISCUSSION TITLE
Locked out from SEB before joining an Exam Provider

### DISCUSSION THREAD

**Discussion by Fruuusty** [SafeExamBrowser/seb-win-refactoring#866] [Q&A]
I recently downloaded and installed SEB from Source Forge, It's my first time using SEB on my own device and I was just testing everything out. When i tried logging out, a prompt popped out asking for an exit password. I put the exit pass as provided by the institution I took a test in, but it wouldn't work.

Now everytime I open SEB it locks me out on a red screen asking me to input the correct exit password, which in this case I still can't figure out the correct one.

Is it possible that the SEB i downloaded from Source Forge was a modded/customized version provided by the institution in which I would have to contact them to know the exact exit pass? Or is there any other explanation regarding to this?

ps : The test will be held on 30th April 2024. Any help would be greatly appreciated!

**Comment by dbuechel** [comment]
You can verify this by checking the application signature, which should be from us (correctly signed by "ETH Zürich"). You could also download the SEB Verificator and verify your installation. If all is okay, then you indeed would need to contact your institution and inquire the correct quit password.
